treeherder/seta/settings.py

```diff
@@ -1,3 +1,9 @@
+# repos that SETA supports
+SETA_PROJECTS = [
+    'mozilla-inbound',
+    'autoland'
+]
```

Comment:
we can have fixed_by_commit for mozilla-central as well- we just don't want to restrict jobs there.